.github/workflows/delete-test-reports.yml

```diff
@@ -0,0 +1,26 @@
+name: Delete Test Report
+
+on:
+  pull_request:
+    types: [closed]
+
+env:
+  BRANCH_NAME: ${{ github.event.pull_request.head.ref }}
+
+jobs:
+  delete-report:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Node
+        uses: ./.github/actions/node
```

Comment:
This seems a bit overkill but I guess there's no other way to get the proper surge version from our package.json 😬

Reply:
Yes 😕